projects/packages/connection/webpack.config.js

```diff
@@ -16,8 +16,57 @@ for ( const file of glob.sync( './src/sso/*.css' ) ) {
 	ssoEntries[ name ].push( path.resolve( file ) );
 }
 
+/**
+ * @type {import('webpack').Configuration[]} Webpack configuration.
+ */
+const sharedConfig = {
+	mode: jetpackWebpackConfig.mode,
+	devtool: jetpackWebpackConfig.devtool,
+	output: {
+		...jetpackWebpackConfig.output,
+		path: path.resolve( './dist' ),
+	},
+	optimization: {
+		...jetpackWebpackConfig.optimization,
+	},
+	resolve: {
+		...jetpackWebpackConfig.resolve,
+	},
+	node: false,
+	module: {
+		strictExportPresence: true,
+		rules: [
+			// Transpile JavaScript, including node_modules.
+			jetpackWebpackConfig.TranspileRule(),
+
+			// Transpile @automattic/jetpack-* in node_modules too.
+			jetpackWebpackConfig.TranspileRule( {
+				includeNodeModules: [ '@automattic/jetpack-' ],
+			} ),
+
+			// Handle CSS.
+			jetpackWebpackConfig.CssRule( {
+				extensions: [ 'css', 'sass', 'scss' ],
+				extraLoaders: [ 'sass-loader' ],
+			} ),
+			// Handle images.
+			jetpackWebpackConfig.FileRule(),
+		],
+	},
+	externals: {
+		...jetpackWebpackConfig.externals,
+		jetpackConfig: JSON.stringify( {
+			consumer_slug: 'identity_crisis',
```

Comment:
We should probably update this to something like
```suggestion
			consumer_slug: 'connection_package',
```
Or move it out of the shared config and into the separate configs below.